styleguide/components/21-atoms/inputs/input-multiselect-chosen/_input-multiselect-chosen.scss

```diff
@@ -1,82 +1,82 @@
-.chosen-container {
-  width: 100% !important; // sass-lint:disable-line no-important
-  max-width: 100% !important; // sass-lint:disable-line no-important
+// .chosen-container {
```

Comment:
Comments

Reply:
Will do so when the designs of all form atoms are final.